elasticsearch/src/main/java/com/gentics/mesh/search/index/AbstractSearchHandler.java

```diff
@@ -212,7 +217,7 @@ public void rawQuery(InternalActionContext ac) {
 
 		RL listResponse = classOfRL.newInstance();
 
-		waitUtil.awaitSync(ac).andThen(Single.defer(() -> {
+		waitUtil.awaitSync(ac).andThen(executeBlocking(vertx, () -> {
```

Comment:
Why was this not changed to singleTx?

Reply:
getSelectedIndices in this lambda already opens a transaction. This lambda does not require a transaction itself, so executeBlocking is used instead.

Reply:
The above rawQuery was now changed to executeBlocking as well.